pkg/cfn/manager/interface.go

```diff
@@ -27,10 +27,12 @@ type UpdateStackOptions struct {
 //counterfeiter:generate -o fakes/fake_stack_manager.go . StackManager
 type StackManager interface {
 	ListNodeGroupStacks() ([]NodeGroupStack, error)
+	ListKarpenterStacks() ([]KarpenterStack, error)
```

Comment:
do we need a `List`? Will we ever have more than 1 for a cluster? `DescribeKarptenterStack` might be enough

Reply:
Probably not. :)